### ISSUE TITLE
Hyperparameter settings for different datasets

### ISSUE THREAD

**Issue by Diqingling** [daiquocnguyen/ConvKB#14]
I tried to use the hyperparameters for FB15K_237 to train NELL_995 but the results (hit-10 42.1%) are far below the benchmarks given in the [KBAT](https://arxiv.org/pdf/1906.01195v1.pdf) paper (hit-10 54.5%). So I want to ask are there any other golden hyperparameter settings for different datasets, such as NELL_995?
Thanks.

**Comment by daiquocnguyen** [comment]
For each dataset, you have to tune the hyper-parameters that work best on the validation set.
